roles/couchdb/Dockerfile

```diff
@@ -1,4 +1,2 @@
-FROM klaemo/couchdb:latest
-
-# Configuration
+FROM klaemo/couchdb:1.6.1
 COPY files /usr/local/etc/couchdb
```

Comment:
yep, if you look at the `volumes` section in couchdb primary, note `/usr/local/lib/npmo/couchdb:/usr/local/var/lib/couchdb`